app/templates/admin.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="row">
-  <div class="sixteen wide column">
+  <div class="x-scrollable">
```

Comment:
Should I nest the div with class x-scrollable within div class=sixteen wide column? Have a doubt about this.